.appveyor.yml

```diff
@@ -0,0 +1,283 @@
+          datalad.local
+          datalad.distributed
+          datalad.support
+          datalad.tests
+          datalad.ui
+      APPVEYOR_BUILD_WORKER_IMAGE: macOS
+      PY: 3.8
+      INSTALL_GITANNEX: git-annex
+      DATALAD_LOCATIONS_SOCKETS: /Users/appveyor/TMP/sockets
+
+
+matrix:
+  allow_failures:
+    - KNOWN2FAIL: 1
+
+
+# tests need specific hostnames to be available
+# note, this is insufficient on MacOS, and needs to be reflected
+# in the SSH config too (tools/ci/appveyor_ssh_config)
+hosts:
+  datalad-test: 127.0.0.1
+  datalad-test2: 127.0.0.1
+
+
+# it is OK to specify paths that may not exist for a particular test run
+cache:
+  # pip cache
+  - C:\Users\appveyor\AppData\Local\pip\Cache -> .appveyor.yml
+  - /home/appveyor/.cache/pip -> .appveyor.yml
+  # TODO: where is the cache on macOS?
+  #- /Users/appveyor/.cache/pip -> .appveyor.yml
+  # TODO: Can we cache `brew`?
+  #- /usr/local/Cellar
+  #- /usr/local/bin
+
+
+# turn of support for MS project build support (not needed)
+build: off
+
+
+# init cannot use any components from the repo, because it runs prior to
+# cloning it
+init:
+  # remove windows 260-char limit on path names
+  - cmd: powershell Set-Itemproperty -path "HKLM:\SYSTEM\CurrentControlSet\Control\FileSystem" -Name LongPathsEnabled -value 1
+  # enable developer mode on windows
+  # this should enable mklink without admin privileges, but it doesn't seem to work
+  #- cmd: powershell tools\ci\appveyor_enable_windevmode.ps1
+  # enable RDP access on windows (RDP password is in appveyor project config)
+  # this is relatively expensive (1-2min), but very convenient to jump into any build at any time
+  - cmd: powershell.exe iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
+  # enable external SSH access to CI worker on all other systems
+  # needs APPVEYOR_SSH_KEY defined in project settings (or environment)
+  - sh: curl -sflL 'https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-ssh.sh' | bash -e -
+  # Identity setup
+  - git config --global user.email "test@appveyor.land"
+  - git config --global user.name "Appveyor Almighty"
+  # Scratch space
+  - cmd: md C:\DLTMP
+  - sh: mkdir /tmp/DL
+  # and use that scratch space to get short paths in test repos
+  # (avoiding length-limits as much as possible)
+  - cmd: "set TMP=C:\\DLTMP"
+  - cmd: "set TEMP=C:\\DLTMP"
+  - sh: export TMPDIR=/tmp/DL
+  # Temporary keys for localhost access in default place
+  - cmd: ssh-keygen -f C:\Users\appveyor\.ssh\id_rsa -N ""
+  - sh: ssh-keygen -f ~/.ssh/id_rsa -N ""
+  # deploy the datalad installer
+  - appveyor DownloadFile https://raw.githubusercontent.com/datalad/datalad-installer/master/src/datalad_installer.py -FileName datalad_installer.py
+
+
+install:
+  # deploy standard SSH config for localhost access on Windows
+  - cmd: tools\ci\appveyor_ssh2localhost.bat
+  - sh: sh tools/ci/appveyor_ssh2localhost.sh
```

Comment:
so it seems that those `ci/appveyor_` aren't really appveyor specific.  Why no to strip `appveyor_` prefix to avoid possible confusion on the purpose?

Reply:
Why do you think that they are not specific? It may be that, right now, nothing special is done in them, but declaring them generic has very limited benefits, and causes entanglement that is undesired.

Reply:
ok